docs/index.md

```diff
@@ -5,19 +5,55 @@ _navPath: toc.html
 _navRel: ../toc.html
 ---
 
-## Getting started
+.NET Documentation
+==================
 
-In this docset, you will find tutorials that will teach you how to get started with .NET Core. You can also find the API reference documentation. 
+Welcome to .NET! You can build any kind of app you want with .NET, from cloud to mobile to desktop to IoT to games. You can start building your next app today, for free, on Windows, OS X and Linux.
```

Comment:
As the new home for .NET docs, I think what you have now under .NET Platform would work well at the top of the page.  Then a Getting Started section that has the app type links under it.  Then you're not immediately sending people away, you are immediately explaining what .NET is and what the implementations are, and your "getting started" is about getting started with your chosen app type.,